### PR TITLE
Added resources for Data Science and Dev community

### PR DESCRIPTION
**Pull request by realsidg**
## Summary:
Added resources for Data Science and Dev community

Added:

- Link for neural network playlist
- Link for Dev community website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,6 +195,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [uxdesign.cc](https://uxdesign.cc/) User Experience, Usability, Product Design
 * [OverAPI.com](http://overapi.com/) Collecting All Cheat Sheets
 * [The list for web designers](https://www.designerslist.info/) A complete toolkit for web designers, wireframes, tutorials, blogs, tools, stock photos, fonts, color schemes, icons...
+* [Dev community](https://dev.to/)
```
Comment (oussamabonnor1) [inline README.md:198]: Please add a small description of this link

File: README.md [hunk 2]
```diff
@@ -141,6 +141,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [Data Science : R Basics](https://courses.edx.org/courses/course-v1:HarvardX+PH125.1x+2T2019): Course about R basics presented by Harvard University
 * [Kaggle: Intro to Machine Learning](https://www.kaggle.com/learn/intro-to-machine-learning): Learn the core ideas in machine learning, and build your first models.
 * [Data quest](https://https://www.dataquest.io/): From intro do advanced data science and analytics content in short lessons.
+* [Neural Networks from Scratch](https://www.youtube.com/watch?v=Wo5dMEP_BbI): A youtube playlist to learn neural networks from scratch using python
```
Comment (oussamabonnor1) [inline README.md:144]: This is only a 15 mn video, you can include the channel in the YouTube section if you want.

File: README.md (hunk 1 above)
Comment (realsidg) [reply README.md]: I have added the description now

File: README.md (hunk 2 above)
Comment (realsidg) [reply README.md]: The link was wrong, I have updated it with the right one. It's a playlist with 3 hrs content.

**Review by oussamabonnor1 — APPROVED** [review]
Wonderful! Thank you for your contribution!

**Pull request merged**